Gui/opensim/installer_files/Linux/INSTALL

```diff
@@ -83,26 +83,31 @@ done
 ## End from https://stackoverflow.com/a/29754866
 
 if [[ -e /etc/os-release ]]; then
-    . /etc/os-release
-    OS=$NAME
-    VER=$VERSION_ID
+    . /etc/os-release 
+    OS=$ID
+    OS_NAME=$NAME
+    if ! [ -z "${VERSION_ID+x}" ]; then
+        VER=$VERSION_ID
+    fi
```

Comment:
My bash skills aren't the greatest (and I wasn't able to figure it out with some quick googling); can you explain what the `+x` is doing? Also, if you're negating the `-z` option, wouldn't using `-n` or even just `[ "$VERSION_ID+x}" ]` be more concise/readable?